### ISSUE TITLE
Installation fails, missing metar

### ISSUE THREAD

**Issue by E3V3A** [supercoderz/pyflightdata#8]
Installation fails with:
```
$ ./arrivals.py -h
Traceback (most recent call last):
  File "./arrivals.py", line 101, in <module>
    from pyflightdata import FlightData
  File "/usr/lib/python3.6/site-packages/pyflightdata/__init__.py", line 1, in <module>
    from .flightdata import FlightData
  File "/usr/lib/python3.6/site-packages/pyflightdata/flightdata.py", line 1, in <module>
    from metar import Metar
ModuleNotFoundError: No module named 'metar'
```

and it doesn't seem to have a PIP.

**Comment by E3V3A** [comment]
```bash
git clone https://github.com/tomp/python-metar.git
cd python-metar
python3 setup.py install
```

**Comment by hari-allamraju** [comment]
Noted, this broke on my server too - working on adding this git dependency to setup.py

Thanks,
Hari

**Comment by hari-allamraju** [comment]
Fixed in 0.5.7 after confirming with library author that version on pypi is stable
